packages/graph/hash_graph/lib/graph/src/ontology/types/serde_shared/array.rs

```diff
@@ -95,4 +115,29 @@ mod tests {
             "additional": 30,
         }));
     }
+
+    mod value_or_array {
+        use serde_json::json;
+
+        use super::*;
+        use crate::ontology::types::serde_shared::tests::{check, StringTypeStruct};
+
+        #[test]
+        fn value() -> Result<(), serde_json::Error> {
+            check(&ValueOrArray::Value("value".to_owned()), json!("value"))
```

Comment:
nit:
```suggestion
            check(
                &ValueOrArray::Value(StringTypeStruct::default()),
                json!({
                    "type": "string"
                }),
            )
```
would be more consistent with the rest of the tests

Reply:
rustfmt I'm afraid